notebooks/en/rag_zephyr_langchain.ipynb

```diff
@@ -1,23 +1,10 @@
 {
```

Comment:
<p>Did you mean to add a code cell here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>